content/docs/introduction/comparison.md

```diff
@@ -204,30 +200,26 @@ once the capacity of a single node is exceeded.
 ### Summary
 
 Prometheus offers a much richer query language, can handle higher cardinality
-metrics and forms part of a complete monitoring system. If you're already
+metrics, and forms part of a complete monitoring system. If you're already
 running Hadoop and value long term storage over these benefits, OpenTSDB is a
 good choice.
 
-
 ## Prometheus vs. Nagios
 
 [Nagios](https://www.nagios.org/) is a monitoring system that originated in the
-90s as NetSaint.
+1990s as NetSaint.
 
 ### Scope
 
 Nagios is primarily about alerting based on the exit codes of scripts. These are called “checks”.
 There is silencing of individual alerts, however no grouping, routing or deduplication.
 
 There are a variety of plugins. For example, piping the few kilobytes of
-perfData plugins are allowed to return [to a time series database such as
-Graphite](https://github.com/shawn-sterling/graphios) or using NRPE to [run
-checks on remote
-machines](https://exchange.nagios.org/directory/Addons/Monitoring-Agents/NRPE--2D-Nagios-Remote-Plugin-Executor/details).
+perfData plugins are allowed to return [to a time series database such as Graphite](https://github.com/shawn-sterling/graphios) or using NRPE to [run checks on remote machines](https://exchange.nagios.org/directory/Addons/Monitoring-Agents/NRPE--2D-Nagios-Remote-Plugin-Executor/details).
 
 ### Data model
 
-Nagios is host-based. Each host can have one or more services, which has one check.
+Nagios is host-based. Each host can have one or more services.
```

Comment:
The checks are a very important part of nagios, they need to be mentioned.

Reply:
Can you clarify what you are trying to say then? The existing didn't make sense.

Reply:
I want to say that there's a one to one mapping between services and checks.

Reply:
Yeah I worked that out - now updated.